### PR TITLE
[WIP] New bundle kit for nukies

### PR DESCRIPTION
**Pull request by OceanFish1**
## About The Pull Request

So this thing adds a bundle kit for nuke ops which includes 5 kits in it that are randomly chosen when you order it with uplink. Those are based on WW2 (maybe not very historical accurate) and each kit has it's unique feature. I started this bundle long ago and I didn't add more countried to it because I don't really want to add that much stuff into one PR. I'll give more descripition about items and guns as  I get closer to making this PR ready for merge. As for now it should remain as it is. Some of these kits can be used for stealth, though most of the abilities will reveal about nukies. Also thanks PigeonVerde for luger sprites.
This kit costs 30 TC and will give you one of 5 kits randomly (they're weighted).

Made a screenshot of how they would originally look like (i'm gonna replace the bandana for the kit on the left to make it look like japanese kamikaze band).
T-shirts are not included.
The spessmen on the right are simple mobs behind the soviet kit.
image

Todo list:

- [ ] Improve that code
- [ ] Feedback
- [x] Fix tommy gun's mag
- [x] Fix stabbing license
- [x] Fix lagging from soviet mobs
- [x] Fix merge conflict
- [ ] Ensure it all works after testing
- [ ] Discuss TC cost politics

## Why It's Good For The Game

More variety in killing the station, gimmick, fun.

## Changelog
:cl:
add: Added new the Old War bundle for nukies, based on whatever something probably close to WW2 I'm not sure.
tweak: Tommy gun now uses it's own inhand sprites instead of shotgun one. It also can use box magazine instead of drum (for balance maybe).
soundadd: added sounds for special items from the kit.
imageadd: added icons for new guns, clothes and items.
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -2328,3 +2337,440 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+/obj/item/stabbing_license/worn_overlays(isinhands)
+	. = ..()
+	if(isinhands)
+		. += mutable_appearance('icons/effects/effects.dmi', "blessed", MOB_LAYER + 0.01)
+		var/obj/item/I
+		if(I.sharpness == IS_SHARP || I.sharpness == IS_SHARP_ACCURATE)
+			. -= mutable_appearance('icons/effects/effects.dmi', "blessed", MOB_LAYER + 0.01)
+			. += mutable_appearance('icons/effects/cult_effects.dmi', "bloodsparkles", MOB_LAYER + 0.01)
+			I.attack_weight = I.attack_weight + 0.5
+			I.force = I.force + 3
```
Comment (PowerfulBacon) [inline code/modules/uplink/uplink_items.dm]: Why is this in the uplink?

Comment (OceanFish1) [reply code/modules/uplink/uplink_items.dm]: I didn't know where else to put it

Comment (OceanFish1) [reply code/modules/uplink/uplink_items.dm]: I guess I should make a new file for them then

Comment (OceanFish1) [reply code/modules/uplink/uplink_items.dm:2776]: nevermind I forgot to add it to devices

**Comment by eeSPee** [comment]
Awesome idea!

**Comment by zeskorion** [comment]
> Those are based on WW2

Mooom the history and gun nerds are making prs agaaiiiiiin

**Comment by OceanFish1** [comment]
conflicts B gone

**Comment by OceanFish1** [comment]
I didn't mean to add rebalance but whatever

**Comment by OceanFish1** [comment]
Closing it for now, will continue later

**Pull request closed without merging**